client/src/frontend/components/resource-slide/Slide/index.js

```diff
@@ -0,0 +1,55 @@
+export * from "./Info";
+export Zoom from "./Zoom";
+
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import Zoom from "./Zoom";
+import { Info } from "./Info";
+
+export default class ResourceListSlide extends Component {
+  static propTypes = {
+    resource: PropTypes.object.isRequired,
+    zoom: PropTypes.bool,
+    zoomLabel: PropTypes.string,
+    info: PropTypes.bool,
+    backgroundUrl: PropTypes.string
+  };
+
+  static defaultProps = {
+    zoom: false,
+    zoomLabel: "Zoom",
+    info: true
+  };
+
+  render() {
+    const {
+      resource,
+      zoom,
+      zoomLabel,
+      children,
+      info,
+      backgroundUrl
+    } = this.props;
+
+    if (!resource) return null;
+
+    const bgImage = backgroundUrl || "/static/images/resource-splash.png";
+
+    let slideStyles = {};
+    if (React.Children.count(children) === 0 || !!backgroundUrl) {
+      slideStyles = { backgroundImage: `url(${bgImage})` };
+    }
+
+    return (
+      <figure>
+        <Zoom show={!!zoom} resource={resource} label={zoomLabel} />
+        <div className="figure-default" style={{ ...slideStyles }}>
+          {info && (
+            <Info resource={resource} withBackground={!!backgroundUrl} />
```

Comment:
Per above, refactor these background checks into method names so that it's super clear what conditions need to be met for this to have a background, and so that the actual check is only happening in one place.